debian/upstream/metadata

```diff
@@ -0,0 +1,5 @@
+---
+Bug-Database: https://github.com/edenhill/kafkacat/issues
```

Comment:
The project has been renamed to kcat, the URLs should be updated accordingly.

Reply:
The bot now follows renames, and has updated this to kcat.